deltacat/io/memcached_object_store.py

```diff
@@ -100,19 +108,19 @@ def _create_ref(self, uid, ip) -> str:
         return f"{uid}{self.SEPARATOR}{ip}"
 
     def _get_client_by_ip(self, ip_address: str):
```

Comment:
I don't see why this method would change. Instead when we are determining IP, if the `storage_node_ips` is passed in the constructor, we use that. Otherwise we can use the current node IP.